sale_elaboration/models/sale_order.py

```diff
@@ -66,14 +66,13 @@ def _compute_route_id(self):
 
     @api.depends("elaboration_ids", "order_id.pricelist_id")
     def _compute_elaboration_price_unit(self):
-        for line in self:
-            elab_price = 0.0
-            for elaboration in line.elaboration_ids:
-                elab_price += elaboration.product_id.with_context(
-                    pricelist=line.order_id.pricelist_id.id,
-                    uom=elaboration.product_id.uom_id.id,
-                ).price
-            line.elaboration_price_unit = elab_price
+        for line in self.filtered(lambda r: r.order_id.pricelist_id):
```

Comment:
Are you sure that doing the filtered don't have missing cache error?
The elaboration_price is not stored.

Reply:
I tested again and I am not getting an error.

Reply:
Oks.. thanks!!

Reply:
I see that this field is stored, so no possibility of miss cache, but I would say this will take `False` value instead of `0.0` with this code. Can you please check?

Reply:
You all right @pedrobaeza , thanks, I corrected it, please check it out.